subprojects/snapshots/src/main/java/org/gradle/internal/snapshot/MerkleDirectorySnapshotBuilder.java

```diff
@@ -78,6 +78,10 @@ public void postVisitDirectory() {
     }
 
     public boolean postVisitDirectory(boolean includeEmpty) {
+        return postVisitDirectory(includeEmpty, false);
```

Comment:
Why is it always not a symlink here?

Reply:
The accessType is now passed always in https://github.com/gradle/gradle/issues/13053.